2048/snapcraft.yaml

```diff
@@ -0,0 +1,22 @@
+name: game-2048
+version: 1
+summary: 2048 game
+description: clone written in pure QML. No C++ wrappers needed.
+confinement: strict
+
+apps:
+  game-2048:
+    command: desktop-launch $SNAP/usr/lib/x86_64-linux-gnu/qt5/bin/qmlscene $SNAP/2048.qml
```

Comment:
QML apps are architecture independent by nature. Can't we drop the `x86_64-linux-gnu` bit so that I can run the app on my x86 laptop? :)